App.js

```diff
@@ -1,26 +1,191 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components/native'
+import { ScrollView } from 'react-native'
+
+const App = () => {
+  const [isRunning, setIsRunning] = useState(false);
+  const [hundredths, setHundredths] = useState(0);
+  const [seconds, setSeconds] = useState(0);
+  const [minutes, setMinutes] = useState(0);
+  const [formattedTime, setFormattedTime] = useState('00:00,00');
+  const [laps, setLaps] = useState([]);
+
+  useEffect(() => {
+    if (isRunning) {
+      const interval = setInterval(() => {
+        setHundredths((hundredths) => {
+          if (hundredths === 99) {
+            setHundredths(0);
+            setSeconds((seconds) => {
+              if (seconds === 59) {
+                setSeconds(0);
+                setMinutes((minutes) => minutes + 1);
+              }
+              return seconds + 1;
+            });
+          }
+          return hundredths + 1;
+        });
+      }, 10);
+      return () => clearInterval(interval);
+    }
+  }, [isRunning]);
```

Comment:
No need to trigger this `useEffect()` each time `isRunning` is changed, interval once initialised will know to execute itself every 10ms